docs/introduction/why-rtk-is-redux-today.md

```diff
@@ -0,0 +1,220 @@
+
+Each of these APIs is completely optional and designed for specific use cases, but are highly recommended.
+
+Note that **Redux Toolkit is still "Redux"!** There's still a single store, with dispatched action objects for updates, and reducers that immutably update state, plus the ability to write thunks for async logic, manage normalized state, type your code with TypeScript, and use the DevTools. **There's just way less code _you_ have to write for the same results!**
+
+## Why We Want You To Use Redux Toolkit
+
+As Redux maintainers, our opinion is:
+
+:::tip
+
+**We want _all_ Redux users to write their Redux code with Redux Toolkit, because it simplifies your code _and_ eliminates many common Redux mistakes and bugs!**
+
+:::
+
+The "boilerplate" and complexity of the early Redux patterns was never a _necessary_ part of Redux. Those patterns only existed because:
+
+- The original "Flux Architecture" used some of those same approaches
+- The early Redux docs showed things like action type constants to enable separating code into different files by type
+- JavaScript is a mutable language by default, and writing immutable updates required manual object spreads and array updates
+- Redux was originally built in just a few weeks and intentionally designed to be just a few API primitives
+
+Over the years, we've seen how people actually used Redux in practice. We've seen how the community wrote hundreds of add-on libraries for tasks like generating action types and creators, async logic and side effects, and data fetching. We've also seen the problems that have consistently caused pain for our users, like accidentally mutating state, writing dozens of lines of code just to make one simple state update, and having trouble tracing how a codebase fits together.
+
+**We specifically designed Redux Toolkit to solve those problems!**
+
+- Redux Toolkit simplifies store setup down to a single clear function call, while retaining the ability to fully configure the store's options if you need to
+- Redux Toolkit eliminates the need to write any action creators or action types by hand
+- Redux Toolkit eliminates the need to write manual and error-prone immutable update logic
+- Redux Toolkit eliminates the need to spread a Redux feature's code across multiple separate files
+- RTK Query eliminates the need to write _any_ thunks, reducers, action creators, or effect hooks to manage fetching data and tracking loading state
+
+Because of this:
+
+:::tip
+
+**We specifically recommend that our users _should_ use Redux Toolkit (the `@reduxjs/toolkit` package), and should _not_ use the legacy `redux` core package for any new Redux code today!**
```

Comment:
```suggestion
**We specifically recommend that our users _should_ use Redux Toolkit (the `@reduxjs/toolkit` package), and should _not_ use the legacy `redux` core package for any new Redux code today!**  
Even for existing applications, we recommend at least switching out `createStore` for `configureStore` as the in-dev middleware will also help you catch accidental mutation errors in existing code bases. We also want to encourage you to switch the reducers you are using most (and any ones you write in the future) over to `createSlice` - while it is a bit of a time invest, we believe that it will safe you a lot of time in the long run.
```